### PR TITLE
[test-only] Api test for folder with dot in name

### PR DESCRIPTION
**Pull request by sakshamgurung**
## Description

- adds api test for webdav operation of folder with dot in the name

## Related Issue

- Test for [2193](https://github.com/owncloud/ocis/issues/2193)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavOperations/deleteFolder.feature [hunk 1]
```diff
@@ -90,3 +90,27 @@ Feature: delete folder
       | dav_version |
       | old         |
       | new         |
+
+  Scenario Outline: deleting folder with dot in the name
+    Given using <dav_version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
```
Comment (jasson99) [inline tests/acceptance/features/apiWebdavOperations/deleteFolder.feature:96]: We already have user `Alice` so I think the creation of user `Brian` is not necessary here. We can perform these actions as user `Alice`.

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareCreateSpecialToShares1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30938/64/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIProxySmoke-3-2-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/30954/169/1

File: tests/acceptance/features/apiWebdavOperations/deleteFolder.feature [hunk 2]
```diff
@@ -90,3 +90,26 @@ Feature: delete folder
       | dav_version |
       | old         |
       | new         |
+
+  Scenario Outline: deleting folder with dot in the name
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/deleteFolder.feature:94]: This runs a lot of individual scenarios, which is good for knowing exactly which unusual folder name causes trouble, and each scenario is short. But it takes time in CI to setup and tear-down each scenario.

**Review by phil-davis — APPROVED** [review]
Looks good - we could refactor the last scenario outline into a single scenario to speed it up, but I guess it's OK.

**Pull request merged**